snapshots/snapshots/src/main/kotlin/com/emergetools/snapshots/compose/SnapshotVariantProvider.kt

```diff
@@ -17,10 +17,18 @@ import androidx.compose.ui.platform.LocalConfiguration
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.platform.LocalLayoutDirection
 import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.LayoutDirection
 import com.emergetools.snapshots.shared.ComposePreviewSnapshotConfig
 import java.util.Locale
 
+private val RTL_LANGUAGES = setOf(
+  "ar", // Arabic
+  "he", // Hebrew (iw is legacy code)
+  "iw",
```

Comment:
There are more languages than this but these are probably the most commonly used for testing. See my other suggestion below which should cover more languages.

Reply:
Yeah intentionally chose to skip the others as these were the primary ones I figured we'd be seeing. I can add all from https://help.smartling.com/hc/en-us/articles/1260802028830-Right-to-left-RTL-Languages, but figured many of these are quite obscure and potentially not worth adding until an explicit request is made to keep things simple

Reply:
Actualy with `TextUtilsCompat` this isn't even necessary!